cmd/plugin/builder/plugin.go

```diff
@@ -112,16 +112,15 @@ func newPluginBuildPackageCmd() *cobra.Command {
 			bppArgs := &plugin.BuildPluginPackageOptions{
 				BinaryArtifactDir:  pbpFlags.BinaryArtifactDir,
 				PackageArtifactDir: pbpFlags.PackageArtifactDir,
-				LocalOCIRegistry:   pbpFlags.LocalOCIRepository,
 				ImgpkgOptions:      imgpkg.NewImgpkgCLIWrapper(),
+				DockerOptions:      docker.NewDockerCLIWrapper(),
 			}
 			return bppArgs.BuildPluginPackages()
 		},
 	}
 
 	pluginBuildPackageCmd.Flags().StringVarP(&pbpFlags.BinaryArtifactDir, "binary-artifacts", "", "./artifacts/plugins", "plugin binary artifact directory")
 	pluginBuildPackageCmd.Flags().StringVarP(&pbpFlags.PackageArtifactDir, "package-artifacts", "", "./artifacts/packages", "plugin package artifacts directory")
-	pluginBuildPackageCmd.Flags().StringVarP(&pbpFlags.LocalOCIRepository, "oci-registry", "", "", "local oci-registry to use for generating packages")
```

Comment:
I believe this is a "breaking" change from the alpha.2 release because it will break users' `plugin-tooling.mk` file

We may want to decide what to do:
1. document the breakage as we have done for alpha.2
2. keep the flag but hidden and just ignore it

Reply:
Thanks for catching this. I have updated the README file.

Regarding the breaking change, I have kept the flag but marked it as hidden for now.